docs/execution-providers/Cloud-ExecutionProvider.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cloud
+description: Instructions to infer an ONNX model remotely with a cloud endpoint
+parent: Execution Providers
+nav_order: 12
+redirect_from: /docs/reference/execution-providers/Cloud-ExecutionProvider
+---
+
+# Cloud Execution Provider
+
+The Cloud Execution Provider enables ONNX Runtime to invoke a cloud endpoint for inferenece, the endpoint must be deployed beforehand.
+Cloud Execution Provider is in preview stage, all API(s) and usage are subjuct to change.
+
+## Contents
+
+To consume the endpoint, ONNX Runtime must load the same model locally, customer could then use configured run options to choose to go remote.
+By far, Cloud Execution Provider is limited to:
+* only support [triton](https://github.com/triton-inference-server) server on [AML](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-with-triton?tabs=python%2Cendpoint).
+* only build and run on Windows and Linux.
```

Comment:
Also, only available as a Python pkg.
Also, this section should be titled ```Limitations (Preview)```

Reply:
Done.